.github/workflows/build.yml

```diff
@@ -39,9 +39,9 @@ jobs:
       - name: Install packages and dependencies
         run: |
           python -m pip install --upgrade pip wheel
-          pip install -e .
+          pip install -e .[test]
           python -c "import autogen"
-          pip install pytest mock
+          pip install mock
```

Comment:
how about keeping line 42 and modifying line 44 into
`pip install pytest-cov mock`?
Only install required packages.